src/controllers/UserController.ts

```diff
@@ -39,6 +40,17 @@ class UserController implements IController{
     const user = await this.userService.create(req.body);
     res.status(201).json({ data: this.userMapper.toDto(user) });
   };
+
+  public updateById: IRequestHandler = async (req, res) => {
+    const user = await this.userService.updateById(Number(req.params.id), req.body);
+    res.status(200).json({ data: this.userMapper.toDto(user) });
+  };
+
+  public deleteById: IRequestHandler = async (req, res) => {
+    const id = Number(req.params.id);
+    await this.userService.deleteById(id);
+    res.status(201).json({ data: `User ${id} was deleted` });
```

Comment:
204!

Reply:
Alright, this had been only corrected in the other branch. Done now in this one.